packages/client/tests/e2e/prisma-config/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../tsconfig.base.json",
+  "compilerOptions": {
+    "skipLibCheck": true
+  },
```

Comment:
Changing this obscures if the fix works or not.

Reply:
`skipLibCheck: true` would "fix" the test on main even without any other changes, and, conversely, this e2e test working with `skipLibCheck: false` would prove the rest of the changes are correct.